components/mpas-ocean/src/Registry.xml

```diff
@@ -1108,14 +1108,26 @@
 					description="If true, topographic wave drag is used on the momentum equation."
 					possible_values=".true. or .false."
 		/>
-		<nml_option name="config_topographic_wave_drag_coeff" type="real" default_value="5.0e-4"
+		<nml_option name="config_topographic_wave_drag_scheme" type="character" default_value="JSL" units="unitless"
+					description="hich of the following three wave drag schemes to use: JSL: Jayne and St. Laurent; ZAE: Zaron and Egbert; or LGF: Local generation formula"
+					possible_values="JSL or ZAE or LGF"
```

Comment:
Is the implementation/tuning for `LGF` well established? I know that `JSL` and especially `ZAE` are working well, but am less confident re: `LGF`.
Should `ZAE` be the default if it's typically the best performing?

Reply:
Having ZAE as the default sounds good to me. I'm going to do a full tidal run with all three and I'll report back on the errors. I believe that `LGF` needs to be re-evaluated given the indexing issue I found above.